doc/introduction/interfaces/torch.rst

```diff
@@ -68,15 +62,31 @@ use different classical interfaces:
         qml.PhaseShift(theta, wires=0)
         return qml.expval(qml.PauliZ(0)), qml.expval(qml.Hadamard(1))
 
-    qnode1 = qml.QNode(circuit2, dev1)
+    qnode1 = qml.QNode(circuit2, dev1, interface='torch')
+
+``qnode1()`` ia now a PyTorch-capable QNode, as well:
+
+>>> qnode1(phi, theta)
+tensor([0.8776, 0.6880], dtype=torch.float64)
+
+Converting an existing QNode
+----------------------------
+
+Sometimes, it is more convenient to instantiate a :class:`~.QNode` object directly, for example,
+if you would like to reuse the same quantum function across multiple devices, or even
+use different classical interfaces:
+
+.. code-block:: python
+
+    dev2 = qml.device('default.mixed', wires=2)
     qnode2 = qml.QNode(circuit2, dev2)
 
 We can convert the default NumPy-interfacing QNode to a PyTorch-interfacing QNode by
 using the :meth:`~.QNode.to_torch` method:
 
->>> qnode1_torch = qnode1.to_torch()
->>> qnode1_torch
-<QNode: device='default.qubit', func=circuit, wires=2, interface=PyTorch>
```

Comment:
Running the cells as they were before doesn't seem to give the `<QNode: device='default.qubit', func=circuit, wires=2, interface=PyTorch>` output. Instead, I get something like `<pennylane.qnode.QNode at 0x7ff01284d370>`.  Do you know the root of this inconsistency?

Reply:
Good catch, we should definitely add the `__repr__` method back to the QNode - it must have gotten lost in a recent refactor. @albi3ro, could you open an issue for this?